dynawo/sources/Modeler/DataInterface/DYNGeneratorInterface.h

```diff
@@ -94,6 +94,12 @@ class GeneratorInterface : public ComponentInterface, public ReactiveCurvePoints
    */
   virtual double getQMax() = 0;
 
+  /**
+   * @brief Getter for the nominal reactive power of the generator diagram
+   * @return The nominal reactive power of the generator diagram in Mvar (generator convention)
```

Comment:
The nominal reactive power of the generator in Mvar (generator convention), computed as the maximum absolute value of the PQ diagram ?